### ISSUE TITLE
Create service class to get bills

### ISSUE THREAD

**Issue by AfroDevGirl** [AfroDevGirl/congross#4]
In a similar pattern as `services/get_representatives.rb` the bill api endpoint will need to be hit, and that data stored in the database. Api information can be found here: https://projects.propublica.org/api-docs/congress-api/bills/#search-bills

**Comment by lao9** [comment]
Hi! I can take a crack at this if the issue is unassigned. :)

**Comment by AfroDevGirl** [comment]
Absolutely! Let me know if you have any questions, the bill data model has a pr open (https://github.com/AfroDevGirl/congross/pull/8) and will be merged shortly.

**Comment by AfroDevGirl** [comment]
@lao9 #8 was just merged and closed

**Comment by lao9** [comment]
Awesome - thanks @AfroDevGirl ! Gonna work on that service tonight. Should have a PR for you to review by tomorrow.

**Comment by lao9** [comment]
Hey @AfroDevGirl - so sorry for the delay here. I've made progress on building my service, but I have a few blockers / questions: 

1. I'm unable to setup my database (can't do `rake db: setup` or anything like that) or run any of the tests (using `bin/rails test`) - I'm running into a config issue because the application.rb file is looking for a config.yml file in config/ directory. Is that something I can create myself, or do you have some guidelines for what you want me to put in there? 

2. For the service - the bill search endpoint can take three params: query, sort, and direction. So far I've accommodated the query by keyword or phrase, but I can also include option sorting params too! 

Let me know when you get a chance!

**Comment by lao9** [comment]
As an update @AfroDevGirl - I was able to circumvent some of the config errors by adding my propublica api key to a config.yml file. I was able to setup my database and run the tests, however now I'm getting an error about my secret_base_key. If you wouldn't mind sharing your recommended config for propublica and secret_base_keys, that'd be very helpful!

I also ran into some issues that I resolved on my feature branch:
-bill fixtures / controller tests are referring to a field that doesn't exist in the db (bill_id) - for a bill model, its id is just 'id'
-committee model was called 'committees' - the corresponding model should be singular

That's all for now! Thanks.

**Comment by AfroDevGirl** [comment]
@lao9 Apologies for missing your updates yesterday, let's setup a time to chat on hangouts or Skype in the next few days. I want to make sure I give you all the right info! You can email me at: amoodyldes at gmail dot com.

**Comment by lao9** [comment]
No problem - sorry for the delay on my end too. Just sent you an email.
